### PR TITLE
K8s-2922: document optional Floating IPs

### PR DESCRIPTION
**Pull request by cbeneke**
Signed-off-by: Christian Beneke <c.beneke@syseleven.de>

### CODE REVIEW

File: pages/02.documentation/06.node-deployments/default.en.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+---
+title: Node Deployments
+published: true
+taxonomy:
+    tag:
+        - nodes
+        - kubernetes
+---
+
+## Node Deployments
```
Comment (bashofmann) [inline pages/02.documentation/06.node-deployments/default.en.md]: This is partly a duplication of https://docs.syseleven.de/metakube/de/documentation/cluster-api

File: pages/02.documentation/06.node-deployments/default.en.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+#### Custom disk (only network storage)
+
+You can choose to use a network volume of custom size for your nodes. Note, that you will be billed for the complete disk, even if the disk size is smaller than the default size.
+
+### Allocate Floating IP
+
+You can choose to allocate floating IPs on all of your nodes. If you do not do so, you need to keep in mind, that you can not directly access your nodes.
+
+!! If you disable the floating IPs for your nodes, you need to have some implications in mind: You can not SSH directly into the nodes, daemonSets which require host-network might not work as expected, etc.
```
Comment (shahbazn) [inline pages/02.documentation/06.node-deployments/default.en.md]: Suggestion to use a jumphost would be nice to add here.

File: pages/02.documentation/06.node-deployments/default.en.md (hunk 1 above)
Comment (shahbazn) [reply pages/02.documentation/06.node-deployments/default.en.md]: Does this need to be updated as well?

File: pages/02.documentation/06.node-deployments/default.en.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+## Node Deployments
+
+MetaKube Clusters consist of master components and worker nodes. The worker nodes are managed through node deployments, which can be added, updated and removed through the API and UI.
+
+When creating a new cluster one node deployment will be added automatically. Nodes within one node deployment are each configured similar.
+
+[This tutorial](../../04.tutorials/09.manage-node-deployments/default.en.md) describes how you can edit the node deployments.
+
+## Settings
```
Comment (shahbazn) [inline pages/02.documentation/06.node-deployments/default.en.md]: Please add a screenshot for this. It is unclear how these settings are being set.

File: pages/02.documentation/06.node-deployments/default.en.md (hunk 1 above)
Comment (cbeneke) [reply pages/02.documentation/06.node-deployments/default.en.md]: Ah, I was searching for that, but didn't find the docs, will merge with that document. We might want to update the tags

File: pages/02.documentation/06.node-deployments/default.en.md (hunk 3 above)
Comment (cbeneke) [reply pages/02.documentation/06.node-deployments/default.en.md]: The linked tutorial already covers how you can access (and where to edit) these settings. We might want to merge this with the cluster-api document anyway

File: pages/02.documentation/12.cluster-api/default.en.md [hunk 4]
```diff
@@ -25,6 +25,50 @@ machineDeployments wrap a rolling update mechanism around machineSets. This mean
...
+
+!! If you disable the floating IPs for your machines, you can not directly access your nodes anymore. Thus you can not SSH directly into the machines, daemonSets which require host-network might not work as expected, etc. To access your nodes via SSH you should create a jumphost in the same network.
+
+#### Image
+
+The image field is pre-filled with the latest image provided by us for the selected operating system. You can choose to use your own image.
+
+#### Node Labels & Taints
+
+You can choose a set of labels and taints, which shall be added to each node of the node deployment. Have a look at [Taints and Tolerations](../10.taints-and-tolerations/default.en.md) for the implications.
```
Comment (mmouselli) [inline pages/02.documentation/12.cluster-api/default.en.md]: I wouldn't use subjunctive. in this case replacing shall with should

File: pages/04.tutorials/09.manage-node-deployments/default.en.md [hunk 5]
```diff
@@ -15,8 +15,7 @@ To add a new node deployment navigate to your cluster view and click on the `Add
 
 ![Cluster overview with highlighted add button](image_add_np_overview.png)
 
-In the popup you can then choose the number of nodes (replicas), kubelet version, flavor, etc for your newly created nodedeployment. All nodes created in this nodedeployment will have the chosen settings.
-
+In the popup you can then choose the number of nodes (replicas), kubelet version, flavor, etc for your newly created nodedeployment. All nodes created in this nodedeployment will have the chosen settings. For a detailed description of the settings have a look at our [cluster-api documentation](../../02.documentation/12.cluster-api/default.en.md).
```
Comment (mmouselli) [inline pages/04.tutorials/09.manage-node-deployments/default.en.md:18]: etc should end with comma

Comment (cbeneke) [reply pages/04.tutorials/09.manage-node-deployments/default.en.md:18]: Wont fix, as we use e.g. etc without punctuation in other tutorials already

File: pages/02.documentation/12.cluster-api/default.en.md (hunk 4 above)
Comment (cbeneke) [reply pages/02.documentation/12.cluster-api/default.en.md]: fixed

**Pull request merged**